docs/source/how-to/deploy-dask-clusters.rst

```diff
@@ -16,7 +28,16 @@ Dask has two families of task schedulers:
     more features, but also requires a bit more effort to set up.  It can
     run locally or distributed across a cluster.
 
-If you import Dask, set up a computation, and then call ``compute``, then you
+|
+
+.. figure:: ../images/dask-overview-distributed-callout.svg
```

Comment:
@jsignell here's the new version of a combined Dask overview & Distributed overview images, based on your suggestion with the magnified map image